utils/server/server.go

```diff
@@ -75,11 +75,11 @@ func New(nodeID, clusterID string, isConsulEnabled, removeProjectScope bool, met
 	e := eventing.New(a, c, fn, adminMan, syncMan)
 
 	c.SetHooks(&model.CrudHooks{
```

Comment:
Pass interface which the eventing module implements. This interface will only have the `Hook` functions